### PR TITLE
xbps-triggers: set homedir of disabled accounts to /var/empty

### PR DESCRIPTION
**Pull request by CameronNemo**
see #5066

### CODE REVIEW

**Comment by maxice8** [comment]
Please update Manual.md accordingly

File: srcpkgs/xbps-triggers/template [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 homepage="http://www.voidlinux.eu"
-license="Simplified BSD"
+license="BSD-2-clause"
 
 do_install() {
 	_triggersdir=usr/libexec/${pkgname}
 
 	for f in ${FILESDIR}/*; do
 		_trigger=$(basename $f)
+		if test "${_trigger}" = "LICENSE"; then
```
Comment (maxice8) [inline srcpkgs/xbps-triggers/template]: i'd rather just download the license with XBPS_FETCH_CMD

Comment (CameronNemo) [reply srcpkgs/xbps-triggers/template]: from where should I download it? if you have an example that would be nice.

Comment (maxice8) [reply srcpkgs/xbps-triggers/template]: Looking at it more i think it is better we just don't worry about it

Comment (CameronNemo) [reply srcpkgs/xbps-triggers/template]: so leave as is, or leave the lint

**Comment by CameronNemo** [comment]
Just did the SPDX change, didn't fetch the license.

**Comment by maxice8** [comment]
If we are updating the homedir when the package is deleted we have to restore it when it is reinstalled.

**Comment by CameronNemo** [comment]
I was under the impression that the shell and homedir is already reset on reinstall, along with other details. I will have to verify this later.

**Comment by maxice8** [comment]
On Mon, Nov 26, 2018 at 08:49:46PM +0000, Cameron Nemo wrote:
> I was under the impression that the shell and homedir is already reset on reinstall, along with other details. I will have to verify this later.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/5069#issuecomment-441792891

No see lines 120 to 132, they check if the account exists in the database and updates the shell and the comment values.

**Comment by CameronNemo** [comment]
Do the primary and / or supplemental groups need to be updated? If a new package version changes the groups that should propagate to existing accounts, right?

**Comment by maxice8** [comment]
makes sense

**Comment by CameronNemo** [comment]
The branch now sets the supplementary groups on update, but I had a hard time wrapping my head around the primary group behavior. Maybe I will look at it later this week, but I am a bit tired today so why bother.

**Comment by maxice8** [comment]
ping

File: srcpkgs/xbps-triggers/template [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'xbps-triggers'
 pkgname=xbps-triggers
-version=0.110
+version=0.111
 revision=1
 noarch=yes
 bootstrap=yes
 short_desc="The XBPS triggers for Void Linux"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="http://www.voidlinux.eu"
```
Comment (maxice8) [inline srcpkgs/xbps-triggers/template]: please set new homepage

**Comment by maxice8** [comment]
request-for-review: @Duncaen

**Pull request merged**